### PR TITLE
Specify the module name in ansible 2.7

### PR DESCRIPTION
**Pull request by jirimacku**
Since the changes in the ovirt_disk will come to ansible 2.8
make sure this version from library folder is going to be used in
ansible 2.7.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by tiraboschi** [comment]
Why do you think we need that?
According to https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#embedding-modules-and-plugins-in-roles

> This can also be used, with some limitations, to modify modules in Ansible’s core distribution, such as to use development versions of modules before they are released in production releases. This is not always advisable as API signatures may change in core components, however, and is not always guaranteed to work. It can be a handy way of carrying a patch against a core module, however, should you have good reason for this. Naturally the project prefers that contributions be directed back to github whenever possible via a pull request.

**Comment by jirimacku** [comment]
The role is still failing with ansible 2.7. Most probably because the original module was used before and the updated module from library folder is not loaded. To rename the module it is a save way how to enforce the module from library to be used.
The same technique was used for example here:
https://github.com/oVirt/ovirt-ansible-vm-infra/pull/43/files by machacekondra.

**Comment by jirimacku** [comment]
ci add to whitelist

File: tasks/create_target_vm/01_create_target_hosted_engine_vm.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@
     register: storage_domain_details
   - debug: var=storage_domain_details
   - name: Add HE disks
-    ovirt_disk:
+    ovirt_disk_27:
```
Comment (tiraboschi) [inline tasks/create_target_vm/01_create_target_hosted_engine_vm.yml]: Can you please rename it as ovirt_disk_28 and add a comment asking to remove it once we will require ansible 2.8?

**Comment by tiraboschi** [comment]
ci build please

**Comment by tiraboschi** [comment]
By default using the standard module name under library/ is enough to have it winning over the vanilla ansible one.
This is just needed if the user is explicitly overriding ANSIBLE_LIBRARY ant env level.

**Pull request merged**